packages/renderer/src/lib/image/ImageDetailsFiles.svelte

```diff
@@ -11,16 +11,17 @@ import FilesystemLayerView from './FilesystemLayerView.svelte';
 import { type ImageFilesystemLayerUI, toImageFilesystemLayerUIs } from './imageDetailsFiles';
 import ImageDetailsFilesLayers from './ImageDetailsFilesLayers.svelte';
 
-export let imageInfo: ImageInfo | undefined;
+let { imageInfo }: { imageInfo: ImageInfo | undefined } = $props();
+
+let imageLayers = $state<ImageFilesystemLayers>();
```

Comment:
$state is used to declare reactive data. Is this required here ?

Reply:
It needs to be a state, as it is modified in line 40 (`imageLayers = layers`) and read in line 67 (`{:else if imageLayers}`)

Reply:
Yes but it is modified in onMount so do we really need reactive here ?

Reply:
It is on a `subscribe` (line 31) initiated in onMount, it can happen at any time